### PR TITLE
Refactored selenium tests and added tests for coverage and edge cases

### PR DESCRIPTION
**Pull request by AntonNiko**

### CODE REVIEW

File: src/tests/seleniumSignUpPage.test.js [hunk 1]
```diff
@@ -150,33 +148,33 @@ describe("User must be able to sign up for an account from the login page and it
...
-        let create_an_account_page = driver.findElement(webdriver.By.xpath("//h2[text()='Create an account']"));
-        await driver.wait(until.elementIsVisible(create_an_account_page), 15000);
+        const create_an_account_page = getElementByXpath(driver, "//h2[text()='Create an account']");
 
-        const email_input_textbox = await driver.findElement(webdriver.By.xpath("//input[@id='validation-sign-up-email']"));
-        const first_name_input_textbox = await driver.findElement(webdriver.By.xpath("//input[@id='validation-sign-up-first-name']"));
-        const password_input_textbox = await driver.findElement(webdriver.By.xpath("//input[@id='validation-sign-up-password']"));
-        const sign_up_form_submit_btn = await driver.findElement(webdriver.By.xpath("//button[text()='Submit']"));
+        const email_input_textbox = await getElementByXpath(driver, "//input[@id='validation-sign-up-email']");;
+        const first_name_input_textbox = await getElementByXpath(driver, "//input[@id='validation-sign-up-first-name']");
```
Comment (skosman) [inline src/tests/seleniumSignUpPage.test.js:158]: super small thing, extra ;

File: src/tests/seleniumSignUpPage.test.js [hunk 2]
```diff
@@ -150,33 +148,33 @@ describe("User must be able to sign up for an account from the login page and it
...
         // Wait until create an account header visible to know we are on sign up page
-        let create_an_account_page = driver.findElement(webdriver.By.xpath("//h2[text()='Create an account']"));
-        await driver.wait(until.elementIsVisible(create_an_account_page), 15000);
+        const create_an_account_page = getElementByXpath(driver, "//h2[text()='Create an account']");
 
-        const email_input_textbox = await driver.findElement(webdriver.By.xpath("//input[@id='validation-sign-up-email']"));
-        const first_name_input_textbox = await driver.findElement(webdriver.By.xpath("//input[@id='validation-sign-up-first-name']"));
-        const password_input_textbox = await driver.findElement(webdriver.By.xpath("//input[@id='validation-sign-up-password']"));
-        const sign_up_form_submit_btn = await driver.findElement(webdriver.By.xpath("//button[text()='Submit']"));
+        const email_input_textbox = await getElementByXpath(driver, "//input[@id='validation-sign-up-email']");;
```
Comment (skosman) [inline src/tests/seleniumSignUpPage.test.js:157]: super small thing, extra ;

**Review by skosman — APPROVED** [review]
Looks good to me!
